src/hmpps/components/court-cases-release-dates/footer/template.njk

```diff
@@ -0,0 +1,96 @@
+                <div class="connect-dps-common-footer__help">
+                    <h2>Support</h2>
+                    <p><a id="footer-support-link" class="connect-dps-common-footer__link" target="_blank" href="{{ params.support.href if params.support.href else 'mailto:calculatereleasedates@digital.justice.co.uk' }}">{{params.support.text if params.support.text else 'Email the specialist support team' }}</a> for help.</p>
+                </div>
+            </div>
+        </div>
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-full">
+                <div class="connect-dps-common-footer__support-links">
+                    <h2 class="govuk-visually-hidden">Support links</h2>
+
+                    <ul class="connect-dps-common-footer__inline-list">
+                        <li class="connect-dps-common-footer__inline-list-item">
+                            <a id="footer-feedback-link" class="connect-dps-common-footer__link" href="{{params.feedback.href if params.feedback.href else 'https://eu.surveymonkey.com/r/Z2MBZZ3' }}" target="_blank" referrerpolicy="no-referrer" rel="noopener">
+                                {{params.feedback.text if params.feedback.text else 'Feedback (opens in a new tab)' }} 
+                            </a>
+                        </li>
+
+                        
+                            <li class="connect-dps-common-footer__inline-list-item">
+                                <a id="footer-accessibility-statement-link" class="connect-dps-common-footer__link" href="{{ params.accessibilityStatement.href if params.accessibilityStatement.href else environmentHrefs[environment].releaseDates + '/accessibility'}}">
+                                    {{params.accessibilityStatement.text if params.accessibilityStatement.text else 'Accessibility' }}
+                                </a>
+                            </li>
+                        
+                            <li class="connect-dps-common-footer__inline-list-item">
+                                <a id="footer-terms-conditions-link" class="connect-dps-common-footer__link" href="{{ params.termsConditions.href if params.termsConditions.href else environmentHrefs[environment].dps + '/terms-and-conditions'}}">
+                                   {{params.termsConditions.text if params.termsConditions.text else 'Terms and conditions'}}
+                                </a>
+                            </li>
+                        
+                            <li class="connect-dps-common-footer__inline-list-item">
+                                <a id="footer-privacy-policy-link" class="connect-dps-common-footer__link" href="{{ params.privacyPolicy.href if params.privacyPolicy.href else environmentHrefs[environment].dps + '/privacy-policy'}}">
+                                    {{params.privacyPolicy.text if params.privacyPolicy.text else 'Privacy policy'}}
```

Comment:
usually my idea formats this in a way where there is a space after `{{`

In intellij I usually do a 'Reformat File'